libs/product/state/src/selectors/composite-product-entities/composite-product-entities.selectors.ts

```diff
@@ -22,12 +22,38 @@ import { DaffProductReducersState } from '../../reducers/product-reducers-state.
 import { getDaffProductEntitiesSelectors } from '../product-entities/product-entities.selectors';
 import { getDaffProductFeatureSelector } from '../product-feature.selector';
 
+/**
+ * An interface for selectors related to the composite product applied options.
+ */
 export interface DaffCompositeProductEntitiesMemoizedSelectors {
+	/**
+	 * The rxjs entities state for composite product applied options.
```

Comment:
rxjs?

Reply:
I was trying to hint that this object is specifically rxjs entity state, which always has the form of `{ entities: [], ids: [] }`. Not a daffodil object.

Reply:
Don't you mean ngrx though? rxjs is just observables

Reply:
yep

Reply:
changed